frontend/homework/11/src/App.tsx

```diff
@@ -0,0 +1,62 @@
+import { useState, useEffect } from "react";
+import { ApiQuotes } from "./types/quotes-types";
+import { Quote } from "./components/Quotes";
+import "./App.scss";
+import { Filter } from "./components/Filter";
+
+function App() {
```

Comment:
Have a main container. quoteContainer component for this. Lets keep App.tsx clean